project1/codesubmission/counterClockwise/Robot.cc

```diff
@@ -0,0 +1,141 @@
+#include "Robot.h"
+#include <cstdlib>
+#include <iostream>
+#include <string>
+#include <limits> // std::numeric_limits
+
+/**Proj1 counter-clockwise: Group10: Aguilar, Andrew, Kamel, Kennedy**/ 
```

Comment:
Once again, just move this over to roomba-roam.cc just in case he's picky about where we put the data.